homeassistant/util/unit_conversion.py

```diff
@@ -288,12 +289,27 @@ class PressureConverter(BaseUnitConverter):
     }
 
 
+class UnitlessConverter(BaseUnitConverter):
```

Comment:
Should we clarify that it's for numbers and not other unitless measurements like eg datetimes?
```suggestion
class UnitlessNumberConverter(BaseUnitConverter):
```

Reply:
Or perhaps:
```suggestion
class UnitlessRatioConverter(BaseUnitConverter):
```

Reply:
I think those are both good suggestions, and I think I prefer `UnitlessRatioConverter` 👍

Reply:
Out of the three I think I like `UnitlessRatioConverter` best.